packages/pigeon/platform_tests/test_plugin/example/ios/RunnerTests/AllDatatypesTests.swift

```diff
@@ -37,7 +37,7 @@ class AllDatatypesTests: XCTestCase {
 
   func testAllEquals() throws {
     let everything = AllNullableTypes(
-      aNullableBool: false,
+      aNullableBool: true,
```

Comment:
were you able to resolve the Bool type issue the other day? What was the fix?

Reply:
switching from Any? to Any

Reply:
Can you provide more details? Not sure which `Any?` you are referring to.

Reply:
it's the same decode issue as the other comments I've replied to